keystrokes/src/components/utils/utils.js

```diff
@@ -0,0 +1,5 @@
+export function getNextKey(givenThis) {
```

Comment:
What the heck is givenThis? 
dont' tell me this is this, please don't do this. if you need to pass in one property looks like keys in this case, ONLY pass in keys. 
AND never modify this, this is mutating your state without you knowing.